docs/deploying-a-database.adoc

```diff
@@ -0,0 +1,242 @@
+=== Deploying a database
+
+This module describes how to deploy a database, connecting to a frontend application.
+
+= Creating a project
+
+.Prerequisites
+
+* {odo-title} is installed.
+* `oc` client.
+* A running OpenShift cluster.
+** Developers can use link:https://cloud.redhat.com/openshift/install/crc/installer-provisioned?intcmp=7013a000002CtetAAC[CodeReady Containers (CRC)] to deploy a local {product-title} cluster quickly.
```

Comment:
Should we not mention Minishift as well since we expect a substantial user base to be on 3.x as well?